src/vs/code/electron-main/sharedProcess.ts

```diff
@@ -61,7 +61,7 @@ export class SharedProcess implements ISharedProcess {
 		};
 
 		const url = `${require.toUrl('vs/code/electron-browser/sharedProcess/sharedProcess.html')}?config=${encodeURIComponent(JSON.stringify(config))}`;
-		this.window.loadURL(url);
+		this.window.loadURL('vscode-file://localhost' + url.slice(7));
```

Comment:
@alexdima @deepak1556 I am a little bit confused about the slicing here, why is `require.toUrl` not returning already with `vscode-file://localhost`?

Reply:
@bpasero the main process is loaded using the `file://` protocol, so the invocation of `require.toUrl` on the main process returns a `file://` url.

Reply:
Oh yeah, true, this is `electron-main`...